joomla/joomla-cms
# [4.0] Fix broken iframe + modal + backdrop combination

Pull Request for Issue #19408 .

### Summary of Changes
Next try for fixing the broken modal and backdrop problem.

The problem (and difference to the article batch which uses the same button than the patchtester) is, that we're loading an IFrame in the modal and doing this, the button creates the modal directly after the button (HTML code). That leads to the fact, that with the position: fixed/relative + flex structure of the page the modal is behind the darkening backdrop background.

A quick solution here is to move the modal in this (and only this) special case just before the ```</body>``` tag, so there is no strange influence from the positioning.

### Testing Instructions
See here: #21343 you can install the patchtester from [here](https://github.com/joomla-extensions/patchtester/releases) and test the "fetch data" modal.
